input/ch.fhir.ig.ch-epr-fhir.xml

```diff
@@ -70,6 +70,11 @@
         <packageId value="ihe.iti.mcsd"/>
         <version value="3.8.0"/>
     </dependsOn>
+    <dependsOn id="ihebalp">
```

Comment:
why do we need to add balp? should that not come automatically?

Reply:
Sushi fails without it, because we directly extend IHE.BasicAudit.Create, IHE.BasicAudit.Update and IHE.BasicAudit.Delete